src/Meta/src/LangChain.csproj

```diff
@@ -13,8 +13,6 @@
         <ProjectReference Include="..\..\Core\src\LangChain.Core.csproj" />
         <ProjectReference Include="..\..\Databases\InMemory\src\LangChain.Databases.InMemory.csproj" />
         <ProjectReference Include="..\..\Extensions\DependencyInjection\src\LangChain.Extensions.DependencyInjection.csproj" />
-        
```

Comment:
Please return Anthropic reference to this project. It's meta package to deliver most used packages.

Reply:
Just added it into the meta